### ISSUE TITLE
Daily grunt work investigation

### ISSUE THREAD

**Issue by zepumph** [phetsims/perennial#408]
@jessegreenberg reported that a few days after changes I made, `grunt update` still showed working copy changes. Daily grunt work should handle this.

**Comment by zepumph** [comment]
Oh my. I was surprised to see that daily-grunt-work actually doesn't run grunt update. That means that the whole codebase is likely a bit off from what tasks like `modulify` expect. Committed now. I'll see how it goes tonight.

**Comment by zepumph** [comment]
I believe everything is looking great. I noticed that the cronjobs are still using node 16, so I updated that, and  few more minor tweaks. I'll check in after tonight again.

- [ ] Turn off page logging for binder doc generation.

**Comment by zepumph** [comment]
@jonathanolson helped me find this article:

**Comment by zepumph** [comment]
That worked excellently! I'll just clean up a couple more errors I see (like the above path), and call it good.

**Comment by zepumph** [comment]
This is the log of all stderr from last nights run:

```
Daily Grunt Work on Thu Nov 21 03:01:12 AM MST 2024
node@v20.18.0, npm@10.8.2
npm warn deprecated osenv@0.1.5: This package is no longer supported.
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated glob@7.1.7: Glob versions prior to v9 are no longer supported
npm warn deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
TASK - COPYRIGHT UPDATES:
To https://github.com/phetsims/bending-light.git
   a7732c8..8df06f1  main -> main
To https://github.com/phetsims/models-of-the-hydrogen-atom.git
   ac2b34e..31e4e57  main -> main
TASK - THIRD PARTY REPORT:
TASK - report third party done, potentially committing
Everything up-to-date
TASK - RESPONSIBLE DEV MARKDOWN:
Everything up-to-date
TASK - GENERATE DATA LISTS:
Everything up-to-date
TASK - REOPEN ISSUES LINKED IN TODOS:
unexpected issue URL: -equations/issues/160
TASK - GRUNT UPDATE:
Everything up-to-date
TASK - BUILD SKIFFLE:
Everything up-to-date
TASK - PUBLISH LATEST PHET_IO LINKS:
TASK - UPDATE LOCALE INFO FILES
Script file not found: ./updateLocaleInfo.js
Everything up-to-date
TASK - BINDER DOC:
npm notice
npm notice New minor version of npm available! 10.8.2 -> 10.9.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.9.0
npm notice To update run: npm install -g npm@10.9.0
npm notice
remote:
remote: GitHub found 11 vulnerabilities on phetsims/binder's default branch (4 critical, 6 high, 1 moderate). To find out more, visit:
remote:      https://github.com/phetsims/binder/security/dependabot
remote:
To https://github.com/phetsims/binder.git
   1f0cc05..ba13082  main -> main
TASK - PUSH ALL CLEANUP:
TASK - DAILY GRUNT WORK COMPLETE

```

Woo!
